.github/workflows/bump-opbeans-node.yml

```diff
@@ -0,0 +1,27 @@
+---
+## Workflow to periodically check if there is an available newer APM agent version, e.g.
+## "1.2.3". If so, then update to it and tag this repo with that version, e.g.
+## "v1.2.3".
+name: Bump opbeans node
+
+on:
+  schedule:
+    - cron:  '0 20 * * 6'
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+          cache: 'npm'
+      - name: Install Updatecli in the runner
+        uses: updatecli/updatecli-action@453502948b442d7b9a923de7b40cc7ce8628505c
+      - name: Run Updatecli
+        run: updatecli diff --config ./.updatecli/bump-opbeans-node.yml
```

Comment:
[updatecli diff](https://www.updatecli.io/docs/commands/updatecli_diff/) show the changes to amake but actually do not perform any other action. [updatecli apply](https://www.updatecli.io/docs/commands/updatecli_apply/) is the one that create PRs

Reply:
https://github.com/elastic/opbeans-node/pull/179/commits/cad12fa86c96c1730edd276d67a403b06c8064bc